tests/jerry/es.next/math-cbrt.js

```diff
@@ -20,6 +20,11 @@ function isSameZero (x, y)
   return x === 0 && (1 / x) === (1 / y);
 }
 
+function isSameDouble (x, y)
```

Comment:
I suggest to use a different name for this function. Perhaps `approxEq` is the shortest name that could be used. It uses "accepted" abbreviations, describes what comparison it performs, and does not conflict with English grammar. (`isSameZero` isn't really a good pattern to follow, unfortunately.)

Reply:
To follow function

function isSameZero (x, y)
{
  return x === 0 && (1 / x) === (1 / y);
}